tests/unit/test_inputs/test_partition_builder.json

```diff
@@ -0,0 +1,1956 @@
+{
+    "PARTITION_FILTERS_LIST": [
```

Comment:
This file is really long... can we adjust to 3-5 partitions instead of 20?

Reply:
Done. Reduced test input size